script.js

```diff
@@ -1,24 +1,14 @@
-//si ij, si i-1j  
-let countgen = 0;
 
-function checkNeighbors() {}
+const container = document.getElementById("container");
```

Comment:
Dont use getElementById, use queryselector

Reply:
Ok Luiz, working on that